go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.6.1
 	github.com/kentik/community_sdk_golang/apiv6 v0.0.0-20210503202321-1dab307d87ff
 )
+
+replace github.com/kentik/community_sdk_golang/apiv6 => github.com/Fenthick/community_sdk_golang/apiv6 v0.0.0-20210531094034-d65e4592bcb6
```

Comment:
If this is related to stub server update (that has been mergerd already), I think it can be removed

Reply:
I'm on it.

Reply:
Actually, already done in other PR.